packages/bpk-component-popover/src/BpkPopoverPortal.js

```diff
@@ -42,21 +44,64 @@ const onOpen = (popoverElement, targetElement) => {
   }
 };
 
-class BpkPopoverPortal extends Component {
+export type Props = {
+  ...$Exact<PopoverProps>,
+  target: Function | Node,
+  isOpen: boolean,
+  onClose: Function,
+  tetherOptions: Object,
+  portalStyle: ?HTMLElement,
```

Comment:
This conflicts with `?Object` above and it isn't really a `HTMLElement` is it?

Reply:
Copypasta! good catch.